### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by jackherizsmith** [fac19/Zaatari-Radio#20]
- [ ] Love how you've coded the Q@A section - could this be abstracted even more? i.e. import an object with question answer key/value pairs to avoid hard-coding them? I suppose only worthwhile if you anticipate having to change or expand these as the app grows

- [ ] React Axe looks cool! Would you recommend?

**Comment by jamesj-0** [comment]
Yeah React Axe has been great, we've had a few accessibility issues with colours mainly so its been useful to see where they are being highlighted. We are guilty of not fixing the errors when they crop up as its been a fast paced build

Also on this note, you should use Eslint! I noticed you'd opted not to use it, It can be quite picky so you can customise its strictness. Would recommend using some configuration of it so everyone in the team is writing consistent code. Just a quick [read](https://medium.com/the-node-js-collection/why-and-how-to-use-eslint-in-your-project-742d0bc61ed7) but expands on this further
